lang/fa/changelogs.php

```diff
@@ -3,9 +3,9 @@
 return [
 	/*
 	|--------------------------------------------------------------------------
-	| Changelogs page
```

Comment:
I don't think we normally translate the section headers? I don't mind, but for consistency. Also the Aspect Ratio one here isn't translated.

Reply:
yeah GPT got a bit overzealous. 😆